src/app/search/directives/search-story.component.ts

```diff
@@ -0,0 +1,100 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { DatePipe } from '@angular/common';
+import { ROUTER_DIRECTIVES } from '@angular/router';
+
+import { HalDoc } from '../../shared/cms/haldoc';
+import { DurationPipe, ImageLoaderComponent, StoryModel } from '../../shared';
+
+@Component({
+  directives: [ImageLoaderComponent, ROUTER_DIRECTIVES],
+  pipes: [DatePipe, DurationPipe],
+  selector: 'publish-search-story',
+  styleUrls: ['search-story.component.css'],
+  template: `
+    <div>
+      <section class="story-image">
+        <image-loader [imageDoc]="story.doc"></image-loader>
+        <p *ngIf="statusClass" [class]="statusClass">{{statusText}}</p>
+      </section>
+      <section class="story-detail">
+        <h2 class="story-title"><a [routerLink]="editStoryLink">{{storyTitle}}</a></h2>
+        
+        <h3 *ngIf="seriesLink" class="series-title"><a [routerLink]="seriesLink">{{seriesTitle}}</a></h3>
+        <h3 *ngIf="!seriesLink" class="series-title">{{seriesTitle}}</h3>
```

Comment:
This doesn't really stand out as the series title to me. It needs something, maybe an icon.